File: Week3_Wed/TableView/UITableView/CodeTableView-OC/CodeTableView-OC/TableViewCell.m
//
//  TableViewCell.m
//  CodeTableView-OC
//
//  Created by kongyunpeng on 1/19/16.
//  Copyright © 2016 kongyunpeng. All rights reserved.
//

#import "TableViewCell.h"

@implementation TableViewCell

-(instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier {
    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
    if (self) {
     //   self.contentView.frame = [UIScreen mainScreen].bounds;
        CGRect rect = CGRectMake(0, 0, [UIScreen mainScreen].bounds.size.width, self.contentView.frame.size.height);
        _label = [[UILabel alloc] initWithFrame:rect];
        
        _label.backgroundColor = [UIColor blueColor];
        _label.textColor = [UIColor whiteColor];
        [self.contentView addSubview:_label];
        
       
    }
    return self;
}

- (void)awakeFromNib {
    // Initialization code
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: Week3_Wed/TableView/UITableView/JKKCodeCell/JKKCodeCell/TableViewCell.swift
//
//  TableViewCell.swift
//  JKKCodeCell
//
//  Created by trainer on 1/20/16.
//  Copyright © 2016 trainer. All rights reserved.
//

import UIKit

class TableViewCell: UITableViewCell {

    var label :UILabel!
    
   override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
      super.init(style: style, reuseIdentifier: reuseIdentifier)
    let width = UIScreen.mainScreen().bounds.size.width
    label = UILabel(frame: CGRect(x: 0, y: 0, width: width, height: self.bounds.size.height))
    
    label.backgroundColor = UIColor.blueColor()
    label.textColor = UIColor.whiteColor()
    self.contentView.addSubview(label)
    
    }

   required init?(coder aDecoder: NSCoder) {
       fatalError("init(coder:) has not been implemented")
   }
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }

}
